sklearn/feature_extraction/text.py

```diff
@@ -681,6 +682,13 @@ def fit(self, X, y=None):
 
         self._validate_params()
 
+        # optional if random_state is left to an integer seed
+        if isinstance(self.random_state, int):
+            self.seed = self.random_state
```

Comment:
same remarks. Plus it should be `self.seed_`

Reply:
`self.seed` is needed here for `self._get_hasher()` below

Reply:
I mean that you should rename seed into seed_ because it's a fitted param. Then in _get_hasher, you create a FeatureHasher instance with random_state=self.seed_